Dockerfile

```diff
@@ -93,6 +97,12 @@ RUN /opt/compiler-explorer/infra/bin/ce_install install 'clang-rocm 5.1.3'
 RUN /opt/compiler-explorer/infra/bin/ce_install install 'clang-rocm 5.2.3'
 RUN /opt/compiler-explorer/infra/bin/ce_install install 'clang-rocm 5.3.2'
 
+# Setup Opam and Install Coq and Menhir for CompCert
+RUN opam init --disable-sandboxing -n \
+    && opam install coq=8.15.2 --yes \
+    && opam install menhir --yes \
+    && eval `opam env`
+
 
 RUN cpanm Modern::Perl
```

Comment:
```suggestion
# Setup for CompCert

# Setup Opam and Install Coq and Menhir for CompCert
RUN opam init --disable-sandboxing -n \
    && opam install coq=8.15.2 --yes \
    && opam install menhir --yes \
    && eval `opam env`

RUN cpanm Modern::Perl
# End of setup for CompCert
```

Reply:
I fixed it

Reply:
Oh, I didn't know you could change directly in my fork.